apps/src/studio/skins.js

```diff
@@ -632,6 +638,12 @@ function loadHoc2015x(skin, assetUrl) {
       timePerFrame: 100
     };
   });
+  skin.bot1.movementAudio = [
+    { begin: 'move1' },
+    { begin: 'move2' },
+    { begin: 'move3' },
+    { begin: 'move4' }
+  ];
```

Comment:
For grid-based movement the one-off effects still seem good - you can do that by just applying "begin" sounds.